calculator.js

```diff
@@ -23,6 +13,73 @@ const calculateUserInput = function (error, promptInput) {
 //   operation: 'add',
 // });
 
+function getValidOperator(operator) {
+const operators = ['+', 'add', '-', 'subtract', '*', 'multiply', '/', 'divide']; 
```

Comment:
Incorrect indentation:

```suggestion
  const operators = ['+', 'add', '-', 'subtract', '*', 'multiply', '/', 'divide']; 
```